test-utils/runtime-tester/src/fuzzing.rs

```diff
@@ -0,0 +1,174 @@
+use crate::run_test::{BlockConfig, NetworkConfig, Scenario, TransactionConfig};
+use near_crypto::{InMemorySigner, KeyType};
+use near_primitives::{
+    account::{AccessKey, AccessKeyPermission},
+    transaction::{Action, AddKeyAction, CreateAccountAction, StakeAction, TransferAction},
+    types::{AccountId, Balance, BlockHeight, Nonce},
+};
+use nearcore::config::{NEAR_BASE, TESTING_INIT_BALANCE};
+
+use libfuzzer_sys::arbitrary::{Arbitrary, Result, Unstructured};
+
+impl Arbitrary<'_> for Scenario {
+    fn arbitrary(u: &mut Unstructured<'_>) -> Result<Self> {
+        let num_accounts = u.int_in_range(2..=100)?;
+
+        let seeds: Vec<String> = (0..num_accounts).map(|i| format!("test{}", i)).collect();
+
+        let mut scope = Scope::from_seeds(&seeds);
+
+        let network_config = NetworkConfig { seeds };
+
+        let num_blocks = u.int_in_range(1500..=2500)?;
```

Comment:
Just realised, that input might not be long enough to create that many blocks. Need to change this to the loop, that runs while u.len() is big enough to create at least an empty block.